django_extensions/management/commands/runserver_plus.py

```diff
@@ -72,7 +72,7 @@ def add_arguments(self, parser):
         parser.add_argument('--browser', action='store_true', dest='open_browser',
                             help='Tells Django to open a browser.')
         parser.add_argument('--nothreading', action='store_false', dest='threaded',
-                            help='Do not run in multithreaded mode.')
+                            help='Do not run in multithreaded mode.', default=None)
```

Comment:
Please use the same style as above and put the `default` kwarg after `dest` instead of `help`.
Thanks

Reply:
Amended.